ipfs-api-prelude/src/api.rs

```diff
@@ -2004,6 +2005,138 @@ pub trait IpfsApi: Backend {
 
     // TODO /pin/verify
 
+    /// Pin object to remote pinning service.
+    ///
+    /// - `service`: Name of the remote pinning service to use (mandatory).
+    ///
+    /// - `name`: An optional name for the pin.
+    ///
+    /// - `background`: Add to the queue on the remote service and
+    /// return immediately (does not wait for pinned status).
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
+    /// use ipfs_api::{IpfsApi, IpfsClient};
```

Comment:
`ipfs_api` has been `deprecated`. Maybe we should not refer to it in our examples?

Reply:
Yeah, I do want to get rid of it eventually just have been lazy about it. You're right that it would definitely be better not to use it anymore in examples.